### ISSUE TITLE
Clarify definitions of audience:Citizen and audience:NonCitizen

### ISSUE THREAD

**Issue by siuc-nate** [CredentialEngine/vocabularies#391]
In order to address issues like trying to indicate when a cost applies to an in-state vs out-of-state vs international student, we need to clarify the definitions of Citizen and NonCitizen so that they consistently apply either to citizens of _states (or provinces, etc.)_ or citizens of _countries_.

Currently, the definition of [Citizen](http://credreg.net/ctdl/vocabs/Audience/Citizen) says:
"A legally recognized subject or national of a state or commonwealth."

And the defintion of [NonCitizen](http://credreg.net/ctdl/vocabs/Audience/NonCitizen) says:
"A person who is not an inhabitant or national of a particular country or other geopolitical region."

The definition for Citizen is confusing, as nobody would be a "national" of a "state or commonwealth". This leads me to believe the intent of Citizen is _probably_ "citizen of a state" - but then the usage note mentions naturalization, which implies "citizen of a country".  

Meanwhile, the definition for NonCitizen explicitly mentions "country" (or other region - presumably a state).  So, it's not clear whether the intent of both is to refer to countries, states, or both.

I believe our original intent was to allow them to apply to either (or both), hence the vague language - but it seems we need to tighten the definitions.

Therefore, we need to rewrite both definitions to ensure clarity and consistency.  To that end, I propose:
**Citizen:** "A person who is a legally recognized subject of a particular state, province, or commonwealth."
**NonCitizen:** "A person who is not a legally recognized subject of a particular state, province, or commonwealth."

My reasoning:
- Use of the full phrase "a person who is/is not" helps with consistency
- Use of "legally recognized subject" is a key aspect
- Use of "of a particular" helps clarify that the person needs to be more than just a citizen of _any_ state
- Use of "state, province, or commonwealth" makes the definition more internationally-friendly without implying "country" (a country is a geopolitical region)

We should also remove the usage note from "Citizen", which says "Includes native or naturalized citizens." because this somewhat implies that Citizen refers to a country, and we are (re)defining it to refer to a state-level entity.

**Comment by stuartasutton** [comment]
These changes are fine, Nate:

**Citizen: **"A person who is a legally recognized subject of a particular state, province, or commonwealth."
**NonCitizen:** "A person who is not a legally recognized subject of a particular state, province, or commonwealth."

**Comment by siuc-nate** [comment]
I still advise removing the usage note from Citizen as well.

Proposed changes:

**Remove:**
> **Subject:** audience:Citizen
> **Predicate:** rdfs:comment
> **Object:** "A legally recognized subject or national of a state or commonwealth."

**Remove:**
> **Subject:** audience:Citizen
> **Predicate:** vann:usageNote
> **Object:** "Includes native or naturalized citizens."

**Remove:** 
> **Subject:** audience:NonCitizen
> **Predicate:** rdfs:comment
> **Object:** "A person who is not an inhabitant or national of a particular country or other geopolitical region."

**Add:**
> **Subject:** audience:Citizen
> **Predicate:** rdfs:comment
> **Object:** "A person who is a legally recognized subject of a particular state, province, or commonwealth."

**Add:** 
> **Subject:** audience:NonCitizen
> **Predicate:** rdfs:comment
> **Object:** "A person who is not a legally recognized subject of a particular state, province, or commonwealth."

**Comment by jkitchensSIUC** [comment]
Typically resident and non-resident are used so is it correct that citizen and non-citizen have the same intent?

**Comment by stuartasutton** [comment]
My sense is that the intention is the same.

**Comment by stuartasutton** [comment]
Should these be changed to "resident" and "non-resident" with the definitions making it clear that they are applicable nation, state, province or comonwealth.

**Comment by siuc-nate** [comment]
That depends on whether the cost applies to residents (regardless of citizenship) or citizens (regardless of residency).  Perhaps we need both?

**Comment by stuartasutton** [comment]
Nate, have you seen examples of both in the wild or are you just speculating?

**Comment by siuc-nate** [comment]
I'm speculating, but I also would rather not deprecate citizen/noncitizen, add resident/nonresident, then later discover we want/need citizen/noncitizen back.  We already have those URIs, so in my opinion we should keep them and add the resident-related ones if we need those.

**Comment by stuartasutton** [comment]
I've no problem with this solution so long as the context of where each applies is clear.

**Comment by siuc-nate** [comment]
Actions to be taken:

**Remove:**
> **Subject:** audience:Citizen
> **Predicate:** rdfs:comment
> **Object:** "A legally recognized subject or national of a state or commonwealth."

**Remove:**
> **Subject:** audience:Citizen
> **Predicate:** vann:usageNote
> **Object:** "Includes native or naturalized citizens."

**Remove:** 
> **Subject:** audience:NonCitizen
> **Predicate:** rdfs:comment
> **Object:** "A person who is not an inhabitant or national of a particular country or other geopolitical region."

**Add:**
> **Subject:** audience:Citizen
> **Predicate:** rdfs:comment
> **Object:** "A person who is a legally recognized subject of a particular state, province, or commonwealth."

**Add:** 
> **Subject:** audience:NonCitizen
> **Predicate:** rdfs:comment
> **Object:** "A person who is not a legally recognized subject of a particular state, province, or commonwealth."

**Add:**
**URI:** audience:Resident
**Label:** Resident
**Definition:** A person whose primary residence is in a particular state, province, or commonwealth.
**In Vocabulary:** ceterms:Audience

**Add:**
**URI:** audience:NonResident
**Label:** Non-Resident
**Definition:** A person whose primary residence is not in a particular state, province, or commonwealth.
**In Vocabulary:** ceterms:Audience

@stuartasutton The last two are new. If those definitions look okay, I can move forward with these.

**Comment by stuartasutton** [comment]
Nate, these were sitting on my machine unsent from this morning. I prefer these defintions.

REMOVE:

>**Subject:** (audience) Citizen
**Predicate:** rdfs:comment 
**Object:** "A legally recognized subject or national of a state or commonwealth."

REMOVE:

>**Subject:** (audience) Citizen
** Predicate:** vann:usageNote 
**Object:** "Includes native or naturalized citizens."

REMOVE:

>**Subject:** (audience) NonCitizen 
**Predicate:** rdfs:comment 
**Object:** "A person who is not an inhabitant or national of a particular country or other geopolitical region."

---------
ADD:

>**Subject:** (audience) Citizen
 **Predicate:** rdfs:comment 
**Object:** "A person who is a legally recognized subject of a particular nation, state, province, or commonwealth."

ADD:

>**Subject:** (audience) NonCitizen 
**Predicate:** rdfs:comment
** Object:** "A person who is not a legally recognized subject of a particular nation, state, province, or commonwealth."

ADD:

>**Subject:** (audience) Resident
 **Predicate:** rdfs:comment 
**Object:** "A non-citizen of a nation, state, province, or commonwealth who has establish a legally recognized residency (domicile).“

ADD:

>**Subject:** (audience) NonResident 
**Predicate:** rdfs:comment
**Object:** "A non-citizen of a nation, state, province, or commonwealth who has not establish a legally recognized residency (domicile).”

**Comment by siuc-nate** [comment]
@stuartasutton Do resident/non-resident need to be constrained to only non-citizens? How would you convey the combination of citizen and non-resident? Or citizen and resident (e.g., a tuition that requires both)?  

I had hoped to avoid potential confusion by deliberately leaving out any reference to citizenship in my proposed definitions for resident/nonresident (just as citizen/non-citizen make no reference to residency), and leave it to those giving us the data to give us the combinations of it that make the most sense to them.

I feel like I'm splitting hairs, but I want the vocabulary to be flexible, and we've used the notion of "picking multiple terms to convey more complex meanings" in other vocabularies as well.  So I see no benefit to having residency and citizenship refer to each other, even if one implies the other in some cases.

**Comment by siuc-nate** [comment]
I've been trying to figure out why we don't seem to quite be connecting on this one. My best guess (and forgive me if I'm off base with any of this) is that I think we may be coming at this from slightly different perspectives - it seems like you're categorizing citizenship and residency along the same scale, e.g.:

`No data` -> `Non-Resident (and noncitizen)` -> `Resident (but noncitizen)` -> `Non-citizen (but resident?)` -> `Citizen (and resident)`

If this is anywhere near correct then I can see why you would want to drop one set and keep the other.

I've been coming at this from a different(?) perspective: citizenship and residency being independent of each other. You can live somewhere (or not). You can be a citizen of somewhere (or not). And what the provider of metadata may want to "say" can be any combination of those, e.g.:

- Resident (citizenship irrelevant)
- Non-resident (citizenship irrelevant)
- Non-resident non-citizen
- Non-resident citizen
- Resident non-citizen
- Resident citizen
- Citizen (residency irrelevant)
- Non-citizen (residency irrelevant)

Hence me wanting to have all four terms so that those giving us the data can indicate which combination their data, constraints, costs, etc., apply to.

In other words, I see citizenship as a legal/governmental "thing" and residency as a physical/geographical "thing". The two may often be used together, but as concepts they are distinct from each other, at least in my mind. In cases where both things need to be indicated, the provider of the metadata would simply select more than one, as with our other vocabularies.

If my guess above isn't correct, then I am at a loss. However, I do come back to the fact that, for better or worse, we already have citizen and non-citizen, so I don't know that we need to drop those two at all, regardless of what we do with resident and non-resident.

**Comment by siuc-nate** [comment]
@stuartasutton @jkitchensSIUC 

Jeanne and I discussed this this morning, and we agree that residency and citizenship are independent concepts that need to be expressed separately in the context of, for example, the costs for a program. We need to finish this one off for the next CTDL release, so it should be a priority for tomorrow's call.

**Comment by siuc-nate** [comment]
Per Jeanne's instructions, I have made the following changes (subject to further changes after we discuss them with Stuart):

REMOVE:

> **Subject:** audience:Citizen
> **Predicate:** rdfs:comment 
> **Object:** "A legally recognized subject or national of a state or commonwealth."

REMOVE:

> **Subject:** audience:Citizen
> ** Predicate:** vann:usageNote 
> **Object:** "Includes native or naturalized citizens."

REMOVE:

> **Subject:** audience:NonCitizen 
> **Predicate:** rdfs:comment 
> **Object:** "A person who is not an inhabitant or national of a particular country or other geopolitical region."

---------
ADD:

>**Subject:** audience:Citizen
 **Predicate:** rdfs:comment 
**Object:** "A person who is a legally recognized subject of a particular nation, state, province, or commonwealth."

ADD:

> **Subject:** audience:NonCitizen 
> **Predicate:** rdfs:comment
> ** Object:** "A person who is not a legally recognized subject of a particular nation, state, province, or commonwealth."

ADD:

> **Subject:** audience:Resident
>  **Predicate:** rdfs:comment 
> **Object:** "A person whose primary residence is in a particular nation, state, province, or commonwealth.“

ADD:

> **Subject:** audience:NonResident 
> **Predicate:** rdfs:comment
> **Object:** "A person whose primary residence is not in a particular nation, state, province, or commonwealth.”

Note: Per Stuart's preference, I included "nation" in the definitions - though this begs the question of how those who provide the data will indicate when they are discussing a national citizenship/residency vs a state citizenship/residency, as things like costs and availability can vary based on that.  I deliberately left "nation" out of my defintions to scope them to states and state-level regions (i.e., Canadian providences) to attempt (perhaps incorrectly) to avoid this confusion.

In other words, since both "nation" and "state" are in the definitions, how would a metadata provider indicate:
- Cost x applies to `resident of [state] and citizen of [state]` (i.e., in-state tuition)
- Cost x applies to `non-resident of [state] and citizen of [nation]` (i.e., out-of-state tuition)
- Cost x applies to `non-residents of [nation] and non-citizens of [nation]` (i.e., commuting or online international tuition)
- Cost x applies to `resident of [state] and non-citizen of [nation]`  (i.e., international students who live on or near campus)

Is this something that requires a reliance on the description field? Or perhaps we should look more at [ceterms:residentOf](http://credreg.net/ctdl/terms/residentOf) (and perhaps add a "citizenOf")?

**Comment by siuc-nate** [comment]
Per our 6/20/2017 discussion:

The changes above are approved, have been made in pending CTDL, and added to history tracking.
